### ISSUE TITLE
Current dark theme feels washed out

### ISSUE THREAD

**Issue by JamesYeoman** [saadeghi/daisyui#571]
For reference, this is how my app looks with the current dark theme
![image](https://user-images.githubusercontent.com/21042602/157709297-b784d10b-af94-4710-8ff3-cfeecf82aec0.png)
and this is how my app looked like with version 2.0.7 of DaisyUI
![image](https://user-images.githubusercontent.com/21042602/157710324-023ccee3-56ba-4c67-abd1-561abe5514bb.png)

I tried creating a new theme using the values from 2.0.7
![image](https://user-images.githubusercontent.com/21042602/157710865-3c726dad-eb65-44e7-ab23-45ecd2f0b0a7.png)

And for the most part, this worked fine. But it doesn't work with the typography tailwind plugin (nothing was getting applied at all, not even header styling).

Is there any plan to reverse this direction with the dark theme? Or at least providing a higher contrast dark theme than in 2.6.4? If not, how would I go about overriding values in the dark theme (because clearly I haven't grabbed all the values needed to fully replace the dark theme)?

**Comment by saadeghi** [comment]
Yes, `dark` theme got darker because it didn't look good on some displays at night (it was too light)
I think it looks better now but if you want the old colors, use this config: https://play.tailwindcss.com/54nC2MwxeG?file=config  
It works good with Typography plugin too. (there were color issues on older versions but it's fixed now)

Let me know if there's an issue
